playbooks/adhoc/s3_registry/s3_registry.yml

```diff
@@ -13,6 +13,8 @@
   vars:
     aws_access_key: "{{ lookup('env', 'S3_ACCESS_KEY_ID') }}"
     aws_secret_key: "{{ lookup('env', 'S3_SECRET_ACCESS_KEY') }}"
+    aws_bucket_name: "{{ aws_bucket | default(clusterid ~ '-docker') }}"
+    aws_bucket_region: "{{ aws_region | default('us-east-1') }}"
```

Comment:
Looks good. We might also want to add a lookup to check and see if the region is already specified in an env variable.

aws_bucket_region: "{{ aws_region | lookup('env', 'EC2_REGION') | default('us-east-1') }}"